app/dashboard/templates/bounty/new.html

```diff
@@ -127,9 +127,28 @@ <h5 class="font-subheader">{% trans "Contact" %}</h5>
                         <label class="form__label" for=hiringRightNow>{% trans "I am hiring for a full time, part time, or contract position, right now and would like Gitcoin to attach this job description to the bounty." %}
                       </label>
                     </div>
-                  <hr class="mt-4">
+                    <input name='jobDescription' id='jobDescription' type="url" class="form__input hidden" type="text" placeholder="https://link.to/job/description" value="" />
```

Comment:
Something is wrong with the behavior of the keywords field on line 81 above... if I click on it the popup appears above the field rather than below as it does in production, and then becomes inactive after a second so you cannot type a keyword in.